registry/tracecat_registry/templates/crowdsec/search_ip_address.yml

```diff
@@ -6,9 +6,9 @@ definition:
   description: Get threat intel report for an IP address from CrowdSec.
   display_group: CrowdSec
   secrets:
-    - name: crowdsec
+    - name: cti
```

Comment:
```suggestion
    - name: crowdsec_cti
```

Reply:
Oh, yes, yes, it's even better actually.